modules/generic_worker/templates/run-generic-worker.sh.multiuser.erb

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# This is a wrapper script for calling generic worker, such that when the worker
+# exits, this script will reboot the machine. Note, in the worker config
+# (/etc/generic-worker.config) we set numberOfTasksToRun to 1 so that the worker
+# will exit after running a single task. This script is then responsible for
+# rebooting the machine.
```

Comment:
This whole paragraph can be removed. It is correct for the simple engine, but for multiuser engine, `numberOfTasksToRun` needs to be `0` so that task users are created for future tasks.